frontend/models/feedback-task-model/record-response/RecordResponse.model.js

```diff
@@ -2,6 +2,7 @@ import { Model } from "@vuex-orm/core";
 
 class RecordResponse extends Model {
   static entity = "recordResponses";
+  static primaryKey = ["id", "question_name"];
```

Comment:
From the API perspective, the `id` is quite enough to define the primary key. But this is only valid if you're using the id from the backend

Reply:
UPDATE: This response represents the response for a single question, so we need id + question (name or id)